src/containers/NdlaFilm/MovieList.jsx

```diff
@@ -0,0 +1,203 @@
+    this.DraggingFile.style.top = 0;
+    this.DraggingFile.style.zIndex = 9999;
+    this.DraggingFile.style.boxShadow = shadows.levitate1;
+    this.DraggingFile.style.transform = `translateY(${this.mouseMovement +
+      MOVIE_HEIGHT}px)`;
+
+    this.setState(
+      {
+        draggingIndex: dragIndex,
+      },
+      () => {
+        // Add transitions
+        Array.from(this.wrapperRef.current.childNodes.values()).forEach(
+          node => {
+            node.style.transition = 'transform 100ms ease';
+          },
+        );
+        this.DraggingFile.style.transition = 'box-shadow 100ms ease';
+      },
+    );
+
+    window.addEventListener('mousemove', this.onDragging);
+    window.addEventListener('mouseup', this.onDragEnd);
+  }
+
+  onDragEnd() {
+    window.removeEventListener('mousemove', this.onDragging);
+    window.removeEventListener('mouseup', this.onDragEnd);
+    const { movies, id } = this.props;
+    // Rearrange movies
+    const toIndex = this.state.draggingIndex;
+    const moveFile = movies[this.initialPosition];
+    movies.splice(this.initialPosition, 1);
+    movies.splice(toIndex, 0, moveFile);
+    this.props.onUpdateMovies(movies, id);
+
+    this.setState({
+      draggingIndex: -1,
+    });
+
+    Array.from(this.wrapperRef.current.childNodes.values()).forEach(
+      (node, index) => {
+        node.style.transition = 'none';
+        node.style.transform = 'none';
+      },
+    );
+
+    this.DraggingFile.style.width = 'auto';
+    this.DraggingFile.style.position = 'static';
+    this.DraggingFile.style.zIndex = 0;
+    this.DraggingFile.style.boxShadow = 'none';
+  }
+
+  onDragging(e) {
+    this.mouseMovement += e.movementY;
+    const currentPosition = Math.max(
+      Math.ceil((this.mouseMovement + MOVIE_HEIGHT / 2) / MOVIE_HEIGHT),
+      0,
+    );
+    const addToPosition = this.initialPosition < currentPosition ? 1 : 0;
+    const dragIndex = Math.min(
+      this.props.movies.length,
+      Math.max(currentPosition, 0),
+    );
+    this.DraggingFile.style.transform = `translateY(${this.mouseMovement +
+      MOVIE_HEIGHT}px)`;
+    this.updateTransforms(dragIndex + addToPosition);
+    if (this.state.draggingIndex !== dragIndex) {
+      this.setState({
+        draggingIndex: dragIndex,
+      });
+    }
+  }
+
+  render() {
+    const { movies, messages } = this.props;
+    const { draggingIndex } = this.state;
+
+    return (
+      <Wrapper>
+        <ListWrapper ref={this.wrapperRef} draggingIndex={draggingIndex}>
+          {movies.map((movie, index) => (
+            <MovieListItem
+              movie={movie}
+              messages={messages}
+              index={index}
+              executeDeleteFile={this.executeDeleteFile}
+              showDragTooltip={movies.length > 1 && draggingIndex === -1}
+              onDragEnd={this.onDragEnd}
+              onDragStart={this.onDragStart}
+              deleteFile={this.deleteFile}
+              {...this.state}
```

Comment:
Bedre å sende inn det som trengs, isteden for å spreade her. Slipper jeg å scrolle opp for å se hva som sendes inn gså.